src/ws/frame.rs

```diff
@@ -372,6 +374,10 @@ impl fmt::Display for Frame {
     }
 }
 
+/// `WebSocket` message with framing.
+#[derive(Debug)]
+pub struct FramedMessage(pub Binary);
```

Comment:
`pub Binary` -> `Binary` 😄

Reply:
```
error[E0611]: field `0` of tuple-struct `ws::frame::FramedMessage` is private
   --> src/ws/client.rs:535:45
    |
535 |             let _ = inner.tx.unbounded_send(data.0.take());
    |                                             ^^^^^^

error[E0611]: field `0` of tuple-struct `ws::frame::FramedMessage` is private
   --> src/ws/context.rs:147:50
    |
147 |             stream.push(ContextFrame::Chunk(Some(data.0)));
    |                                                  ^^^^^^
```

Reply:
I could make it `pub(crate)` though if that helps?

Reply:
Oh I see, sorry, I didn't notice it is used outside of this module.
Yes, it should be public only within crate itself, we cannot let user to write non-framed data.

Reply:
done